code/modules/mob/living/simple_animal/hostile/terror_spiders/terror_spiders.dm

```diff
@@ -41,6 +41,7 @@ var/global/list/ts_spiderling_list = list()
 	pressure_resistance = 50    //50 kPa difference required to push
 	throw_pressure_limit = 100  //100 kPa difference required to throw
 	pass_flags = PASSTABLE
+	anchored = 1 // cannot be dragged
```

Comment:
`anchored` does a lot more than just removing draggability - it makes them unable to be pushed, teleported, or subject to any other effect that deliberately omits "furniture". Because of the manner of its use throughout the codebase is specific towards "furniture" objects, I'm skeptical about giving it to a walking and moving mob - the only other mob I know of that has `anchored` set being the station AI. I'd instead suggest adding another var or proc checked inside of [`start_pulling`](https://github.com/ParadiseSS13/Paradise/blob/57a07815f91e06a6cb58bdf40dd9d54704e036bf/code/modules/mob/pulling.dm#L2)